doc/version_history.rst

```diff
@@ -2,7 +2,14 @@
 Version History
```

Comment:
Is that really needed? Why? There are tags for that..

Reply:
I only see two tags in this repository... I just updated this file because I saw this file. Or do you want me to remove the update in this file?

Reply:
Those are minor changes, don't change anything in logic. Those are nice, but don't require new version, as this doesn't bring any new feature. Reformatting will shown in git log, we will do new version once we have new features/bug fixes - this just grow version log, which nobody will read anyway, as you basically bisec through tags if you discover a problem.